sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -441,7 +441,7 @@ std::pair<RT::PiKernel, std::mutex *> ProgramManager::getOrCreateKernel(
       [&Program](const Locked<KernelCacheT> &LockedCache) -> KernelByNameT & {
     return LockedCache.get()[Program];
   };
-  auto BuildF = [this, &Program, &KernelName, &Ctx] {
+  auto BuildF = [&Program, &KernelName, &Ctx] {
```

Comment:
This is to correct this:
program_manager.cpp:  error: lambda capture 'this' is not used [-Werror,-Wunused-lambda-capture]
   auto BuildF = [this, &Program, &KernelName, &Ctx] {